app/services/users/default_categories_creator.rb

```diff
@@ -0,0 +1,17 @@
+class Users::DefaultCategoriesCreator < Struct.new(:user)
+  def call
+    default_categories_list.map { |title| create_category(title) }
+  end
+
+  private
+
+  def create_category(title)
+    user_categories.create(title: title, amount: 0, user_id: user.id)
```

Comment:
@marderer you don't have to pass `user_id` because you are creating categories from a user association

Reply:
@AleksSenkou and here :(